### PR TITLE
Update remote_provider.go

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

### CODE REVIEW

File: models/remote_provider.go [hunk 1]
```diff
@@ -124,6 +124,7 @@ func (l *RemoteProvider) loadCapabilities(token string) {
 	if err := decoder.Decode(&l.ProviderProperties); err != nil {
 		logrus.Errorf("[Initialize]: Failed to decode provider properties %s", err)
 	}
+	l.ProviderProperties.PackageVersion = version;
```
Comment (warunicorn19) [inline models/remote_provider.go:127]:
```suggestion
	l.ProviderProperties.PackageVersion = version
```

**Comment by leecalcote** [comment]
I'll put this PR on-hold pending @sudo-NithishKarthik's investigations.

**Comment by sayantan1413** [comment]
Yes, @leecalcote I think Nithish has figured out a way in Meshery-Cloud. Let's try that out first.

**Comment by warunicorn19** [comment]
Yup let's try that out, I was just fixing the golangci-lint check for now.

**Pull request closed without merging**